tests/e2e-upgrade/upgrade/.gitignore

```diff
@@ -0,0 +1,2 @@
+00-setup.yaml
```

Comment:
Because this file is generated from `00-setup.yaml.envsubst`.

Unfortunately kuttl doesn't allow any parameterization of tests, therefore I made this workaround with envsubst.